server/src/__tests__/fixtures/fixtures.ts

```diff
@@ -4,8 +4,8 @@
  * ------------------------------------------------------------------------------------------ */
 
 /**
- * Inspired by bash-language-sever
- * Repo: https://github.com/bash-lsp/bash-language-server
+ * Inspired by bash-language-server under MIT
+ * Reference: https://github.com/bash-lsp/bash-language-server/blob/main/testing/fixtures.ts
```

Comment:
This one points to "main" which can be updated in the future. You may consider using a fixed commit like your other references.

Reply:
Done, now it uses one with a fixed commit in the link.